{"uuid": "1f45d910-2be1-49d7-ab34-15b81222d5f9", "children": ["b3e759a4-6aa3-44af-b6cd-9388ea1d9bfd", "020b41d9-eda6-488a-8173-9642a63a120f"], "befores": [{"name": "create_user", "status": "broken", "statusDetails": {"message": "requests.exceptions.SSLError: HTTPSConnectionPool(host='stellarburgers.nomoreparties.site', port=443): Max retries exceeded with url: /api/auth/register (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate in certificate chain (_ssl.c:1000)')))\n", "trace": "  File \"C:\\Users\\nadezhda.salina\\Desktop\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Users\\nadezhda.salina\\Desktop\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\nadezhda.salina\\Desktop\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\nadezhda.salina\\Desktop\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\nadezhda.salina\\Desktop\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Users\\nadezhda.salina\\Desktop\\Diplom\\Diplom_3\\conftest.py\", line 31, in create_user\n    user_creation = ApiShop.create_user(user_data)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\nadezhda.salina\\Desktop\\Diplom\\Diplom_3\\api_shop.py\", line 9, in create_user\n    return requests.post(f'{Url.MAIN_URL}{ApiUrls.CREATE_USER}', json=body)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\nadezhda.salina\\Desktop\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\requests\\api.py\", line 115, in post\n    return request(\"post\", url, data=data, json=json, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\nadezhda.salina\\Desktop\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\requests\\api.py\", line 59, in request\n    return session.request(method=method, url=url, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\nadezhda.salina\\Desktop\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\requests\\sessions.py\", line 589, in request\n    resp = self.send(prep, **send_kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\nadezhda.salina\\Desktop\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\requests\\sessions.py\", line 703, in send\n    r = adapter.send(request, **kwargs)\n        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\nadezhda.salina\\Desktop\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\requests\\adapters.py\", line 698, in send\n    raise SSLError(e, request=request)\n"}, "start": 1732722256288, "stop": 1732722257528}], "start": 1732722256288, "stop": 1732722257754}